contrib/generate_precompile.jl

```diff
@@ -126,6 +126,11 @@ function generate_precompile_statements()
             m === nothing && continue
             statement = m.captures[1]
             occursin(r"Main.", statement) && continue
+            for (l, r) in (('(', '{'), (')', '}'))
+                if count(isequal(l), statement) != count(isequal(r), statement)
+                    continue
```

Comment:
I think you want to break out of the outer `for` loop — this will just skip to the next `(l, r)` test.  Might as well just write the two cases out by hand.

Reply:
Yeah, of course, sorry. Will fix.